src/omdata.h

```diff
@@ -22,6 +22,27 @@ struct overmap_spawns {
     int max_population;
     int chance;
 };
+ //terrain flags enum! this is for tracking the indices of each flag.
+    //is_asphalt, is_building, is_subway, is_sewer, is_ants,
```

Comment:
Agreed that in this context the comment-listing is unhelpful.  Something along the lines of "add new terrain flags to this list" (presuming that that is in fact the case) would likely be more helpful.